### ISSUE TITLE
[SOLVED] This ORT build has ['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider']

### ISSUE THREAD

**Issue by mr-bob-chang** [Gourieff/comfyui-reactor-node#294]
### First, confirm

- [X] I have read the [instruction](https://github.com/Gourieff/comfyui-reactor-node/blob/main/README.md) carefully
- [X] I have searched the existing issues
- [X] I have updated the extension to the latest version

### What happened?

There was an error I don't understand, Something about ORT 1.9.

### Steps to reproduce the problem

Your workflow
[FaceSwapErrorWorkflow.json](https://github.com/Gourieff/comfyui-reactor-node/files/15157631/FaceSwapErrorWorkflow.json)

### Sysinfo

I work on Windows11, Edge browser, RTX 4090

### Relevant console log

```Shell
Error occurred when executing ReActorFaceSwap:

This ORT build has ['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider'] enabled. Since ORT 1.9, you are required to explicitly set the providers parameter when instantiating InferenceSession. For example, onnxruntime.InferenceSession(..., providers=['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider'], ...)

File "E:\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "E:\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "E:\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node\nodes.py", line 310, in execute
script.process(
File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node\scripts\reactor_faceswap.py", line 91, in process
result = swap_face(
^^^^^^^^^^
File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node\scripts\reactor_swapper.py", line 226, in swap_face
source_faces = analyze_faces(source_img)
^^^^^^^^^^^^^^^^^^^^^^^^^
File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node\scripts\reactor_swapper.py", line 142, in analyze_faces
face_analyser = copy.deepcopy(getAnalysisModel())
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "copy.py", line 172, in deepcopy
File "copy.py", line 271, in _reconstruct
File "copy.py", line 146, in deepcopy
File "copy.py", line 231, in _deepcopy_dict
File "copy.py", line 146, in deepcopy
File "copy.py", line 231, in _deepcopy_dict
File "copy.py", line 172, in deepcopy
File "copy.py", line 271, in _reconstruct
File "copy.py", line 146, in deepcopy
File "copy.py", line 231, in _deepcopy_dict
File "copy.py", line 172, in deepcopy
File "copy.py", line 273, in _reconstruct
File "E:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\insightface\model_zoo\model_zoo.py", line 33, in __setstate__
self.__init__(model_path)
File "E:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\insightface\model_zoo\model_zoo.py", line 25, in __init__
super().__init__(model_path, **kwargs)
File "E:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\onnxruntime\capi\onnxruntime_inference_collection.py", line 396, in __init__
raise e
File "E:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\onnxruntime\capi\onnxruntime_inference_collection.py", line 383, in __init__
self._create_inference_session(providers, provider_options, disabled_optimizers)
File "E:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\onnxruntime\capi\onnxruntime_inference_collection.py", line 415, in _create_inference_session
raise ValueError(
```

**Comment by sonofdbn** [comment]
I had the same problem. This fixed it for me: https://github.com/Gourieff/comfyui-reactor-node/discussions/73#discussioncomment-7317684

**Comment by mr-bob-chang** [comment]
> I had the same problem. This fixed it for me: [#73 (comment)](https://github.com/Gourieff/comfyui-reactor-node/discussions/73#discussioncomment-7317684)

Thank you so much, problem solved!
